### DISCUSSION TITLE
YOLO Model Fine-tuning

### DISCUSSION THREAD

**Discussion by FrozenPeak0701** [infstellar/genshin_impact_assistant#113] [General]
Hi, I saw that you used YOLO models for the image detection module. Did you guys use a pre-trained version or finetuned the pre-trained model using COCO dataset or just trained from scratch using COCO or actually have a synthetic dataset of Genshin images?

 I am currently doing a class project on finetuning yolo/ssd models for monster detection in Genshin in-game scenes and I intend to use Genshin images to fine-tune a pre-trained model. However, manually annotating screenshots would be too much of a hassle. I wonder if there's an existing dataset somewhere, or if there's a way to hack the game and get annotations like bounding boxes and labels and segmentations easily. Any thoughts?
